duo_wordpress.php

```diff
@@ -468,6 +485,13 @@ function duo_update_mu_options() {
         else {
             $result = update_site_option('duo_xmlrpc', 'on');
         }
+        if(isset($_POST['duo_rest'])) {
+            $rest = sanitize_text_field($_POST['duo_rest']);
+            $result = update_site_option('duo_rest', $rest);
```

Comment:
This should probably just read "off" instead of `$rest` to avoid having to sanitize the text field at all (on the line above) and avoid ever ending up in a third state (neither "off" nor "on").